### PR TITLE
+session.0.5.0

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The tests fail with:
```
#=== ERROR while compiling session.0.5.0 ======================================#
# context              2.0.9 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/inhabitedtype/ocaml-session/archive/0.5.0.tar.gz)
# path                 ~/.opam/4.07/.opam-switch/build/session.0.5.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p session -j 31
# exit-code            1
# env-file             ~/.opam/log/session-654-ffb3fd.env
# output-file          ~/.opam/log/session-654-ffb3fd.out
### output ###
#     ocamlopt lib_test/memory/test.exe (exit 2)
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlopt.opt -w -40 -g -o lib_test/memory/test.exe /home/opam/.opam/4.07/lib/bigarray-compat/bigarray_compat.cmxa /home/opam/.opam/4.07/lib/cstruct/cstruct.cmxa -I /home/opam/.opam/4.07/lib/cstruct /home/opam/.opam/4.07/lib/ocplib-endian/ocplib_endian.cmxa /home/opam/.opam/4.07/lib/mirage-crypto/mirage_crypto.cmxa -I /home/opam/.opam/4.07/lib/mirage-crypto /home/opam/.opam/4.07/lib/mirage-crypto-rng/mirage_crypto_rng.cmxa /home/opam/.opam/4.07/lib/base64/base64.cmxa lib/session.cmxa /home/opam/.opam/4.07/lib/ocaml/unix.cmxa -I /home/opam/.opam/4.07/lib/ocaml /home/opam/.opam/4.07/lib/stdlib-shims/stdlib_shims.cmxa /home/opam/.opam/4.07/lib/ounit2/advanced/oUnitAdvanced.cmxa /home/opam/.opam/4.07/lib/ounit2/oUnit.cmxa /home/opam/.opam/4.07/lib/mirage-crypto-rng/unix/mirage_crypto_rng_unix.cmxa -I /home/opam/.opam/4.07/lib/mirage-crypto-rng/unix lib_test/session_test.cmxa lib_test/memory/.test.eobjs/native/test.cmx)
# File "_none_", line 1:
# Error: No implementations provided for the following modules:
#          Unix referenced from lib/session.cmxa(Session__Memory)
```

File: packages/session/session.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "dune" {>= "1.0"}
+  "mirage-crypto"
+  "mirage-crypto-rng"
+  "base64" {>= "3.1.0"}
+  "ounit" {with-test & >= "1.0.2"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Comment (talex5) [inline packages/session/session.0.5.0/opam:19]:
```suggestion
```

The upstream project's Travis configuration doesn't run the tests for this package (and they don't link).

**Review by talex5 — COMMENTED** [review]
Also, looks like one of the older releases has changed its checksum (session-cookie-lwt-0.1.8.tbz):
```
The archive doesn't match checksums:
- archive: sha256=d5e02adf527b49ae77ed57bdf0bd68e3a059ad66ed0b07cf33f77e8ac2782c75, in opam file:
           sha256=bc405a88a6f94453fbaee19b98469818c09b567646a46e1f4ac1a9d5d9fee2db
- archive: sha512=7c40d318b9186a3747d2ed0e75d64179e0f7fcee1b00710f0a1bfda6f1f5af921fe532c810796559b30e32bd8e3cdb27ca0d2fe626d82128c4fae2c67059419a, in opam file: 
           sha512=fd66a50db8ca8431e7e0ec847c828f448a61b7fbfea9581a75b2b93c205ec4728b072fa2bdc6c70e7afe41f6e6254756fb176ce9c3f71a0cb6da0086479644ed
```

**Comment by avsm** [comment]
cc @ulrikstrid -- would you know why session-cookie-lwt.0.1.8 has changed its checksum?

**Comment by ulrikstrid** [comment]
I have not changed it as far as I know

**Comment by ulrikstrid** [comment]
image
It seems like they have different names in the top folder. Other than that I can't see any differences

**Comment by kit-ty-kate** [comment]
Thanks sorry for the delay

**Pull request merged**